### ISSUE TITLE
aria-modal not supported

### ISSUE THREAD

**Issue by sophieschuermans** [FreedomScientific/VFO-standards-support#91]
### Summary

aria-modal is supposed to prevent the screenreader to access content out of the modal window, but this does not work in the following example:
1. Go to http://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html
2. Go to the first button, 'Add delivery address'
3. Hit ENTER
4. Use the arrow up key several times to attempt to go out of the modal dialog.

### Expected result
It should not be possible to interact with any content outside of the modal dialog

### Actual result
Using the arrow key, it is possible to go to the content of the page that is behind the modal dialog. 

### Example
See above

#### JAWS version and build number
JAWS 18.0.5038

 
#### Operating System and version
Windows 10

#### Browser and version:
Internet explorer 11

**Comment by cowell-cor** [comment]
@sophieschuermans  did you get any updates on this?

**Comment by sophieschuermans** [comment]
No, I didn't get any updates.

**Comment by stevefaulkner** [comment]
@sophieschuermans I have commented on an existing internal bug: 95129 - Internet Explorer: JAWS users can access background content when modal window is active, with the bug report here, thanks for filing!

**Comment by kbunge63** [comment]
Excellent info and perfectly timed, I just rant into this issue 5 minutes ago in a review!

**Comment by scottaohara** [comment]
Update:

Navigating specifically with arrow keys and other hot keys is trapped within modal dialogs using `aria-modal="true"`.

There are still some quirks with `aria-modal`, but they fall outside the original scope of this issue.

I have logged them in #179, and am closing this issue per the described issue here having been rectified.

**Comment by JAWS-test** [comment]
The problem with aria-modal is not solved in IE 11. In the example https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html aria-modal=true or aria-modal=false has no effect on the output. Decisive is the role=dialog, which limits linear reading to the pop-up. If role=dialog is removed, the whole page can also be read with aria-modal=true. Therefore the ticket should be opened again.
